dist_run.py

```diff
@@ -184,6 +187,13 @@ def main():
     logger.info(f"Loading weights for {pp_rank=} on {device=}")
     _load_model_weights(model, hf_model_name, device=device, model_config=config)
 
+    # info on stage size and params
+    stage_size, stage_size_formatted = get_stage_size(model)
```

Comment:
nit:
can we make the API return just a number? At the place of logging, we can then use `bytes_to_readable` to get a string.

Reply:
updated